wordpress-mobile/WordPress-Android
# Quick start tasks

Part of #7707

This PR adds actual Quick Start tasks and allows you to complete them as long as you are followinf the instructions.

I opted to use EvenBus to reduce the coupling and extra boilerplate code required to pass tasks to fragments.

To test:

1. Navigate to Quick Start activity from My Site fragment.
2. Click on each tutorial, make sure the directions and visual clues of tasks are correct, and that following them results in a completion of a task (task will be marked as completed in Quick Start list).
